privacyIDEA Authenticator.xcodeproj/project.pbxproj

```diff
@@ -825,16 +809,16 @@
 			buildSettings = {
 				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
 				CODE_SIGN_ENTITLEMENTS = "privacyIDEA Authenticator/privacyIDEA Authenticator.entitlements";
-				CODE_SIGN_IDENTITY = "iPhone Developer";
+				CODE_SIGN_IDENTITY = "Apple Development";
 				CODE_SIGN_STYLE = Automatic;
-				DEVELOPMENT_TEAM = 627QALYL3B;
+				DEVELOPMENT_TEAM = 52NSLTN3S4;
 				INFOPLIST_FILE = "privacyIDEA Authenticator/Info.plist";
 				IPHONEOS_DEPLOYMENT_TARGET = 12.4;
 				LD_RUNPATH_SEARCH_PATHS = (
 					"$(inherited)",
 					"@executable_path/Frameworks",
 				);
-				PRODUCT_BUNDLE_IDENTIFIER = privacyidea.authenticator;
+				PRODUCT_BUNDLE_IDENTIFIER = net.cowism.privacyidea.authenticator;
```

Comment:
we probably don't want a changed identifier

Reply:
Yeah, sorry, I had to change that to get it to compile and forgot to revert/cherry-pick the change.